samples/sample/src/commonMain/kotlin/com/careem/mockingbird/sample/PippoSample.kt

```diff
@@ -24,4 +24,5 @@ interface PippoSample {
     fun sayHiWith(param: String)
     fun sayHiWith(param: String, someOtherParam: Boolean)
     fun sayHiWithCommonParam(param: String, intParam: Int)
+    suspend fun thisIsSuspend(param: String, intParam: Int)
```

Comment:
I dont see u have any tests for this

Reply:
we are going to write a test for code generation and will include this one as well

Reply:
#95